plugins/filter/crc32.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022, Julien Riou <julien@riou.xyz>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from ansible.errors import AnsibleFilterError
+from ansible.module_utils.six import string_types
+from ansible.module_utils.common.text.converters import to_bytes
+
+try:
+    from zlib import crc32
+    HAS_ZLIB = True
+except ImportError:
+    HAS_ZLIB = False
+
+
+DOCUMENTATION = '''
+  name: crc32
+  short_description: Generate a CRC32 checksum
+  version_added: 5.4.0
```

Comment:
I'm not sure about this version. Could you please advice what version should I put here please?

Reply:
5.4.0 sounds good, that will be the next minor release (in around two weeks). It's quite realistic that this will be merged by then.